lib/dateDisplay.ml

```diff
@@ -20,37 +20,46 @@ let death_symbol conf =
   try List.assoc "death_symbol" conf.base_env
   with Not_found -> "†"
 
-let code_date conf encoding d m y =
```

Comment:
It's not neefed anymore to split code_date in gen_code_date/code_date.

Please restore this part so the diff we be only what is necessary.